### PR TITLE
Added header button in accordion for watch expressions (#916)

### PR DESCRIPTION
**Pull request by jbhoosreddy**
Associated Issue: #916 
### Summary of Changes
- Added header button for displaying the watch expression input.
  For now I used a placeholder icon instead of the `+` icon.
### Testing
- [x] passes `npm test`
- [x] passes `npm run lint`
### Screenshots/Videos (OPTIONAL)

There are two new buttons in the header. The icons is temporary but, the first one is to toggle expression input while the second is to refresh the expressions. 
screen shot 2016-10-17 at 3 55 20 am

### CODE REVIEW

File: public/js/actions/pause.js [hunk 1]
```diff
@@ -206,6 +206,15 @@ function addExpression(expression) {
   };
 }
 
+function setExpressionInputVisibility(visibility) {
```
Comment (jasonLaster) [inline public/js/actions/pause.js]: Hmm, at this point we're opting to use component state local UI data. Some examples of this are, the search bar value, app search being shown, ... 

I think at some point, we might want to start using redux state more for UI data like this, but for the moment it is simpler to use component state.

File: public/js/components/RightSidebar.css [hunk 2]
```diff
@@ -68,3 +68,7 @@ span.pause-exceptions.uncaught {
 span.pause-exceptions.all {
   stroke: var(--theme-highlight-blue);
 }
+
+.accordion-button {
+  float: right;
+}
```
Comment (jasonLaster) [inline public/js/components/RightSidebar.css]: This works, but it might be nice to try flex as well. An example of that is the autocomplete layout.

**Comment by wldcordeiro** [comment]
Screenshots please @jbhoosreddy? 😄

**Comment by jbhoosreddy** [comment]
@wldcordeiro I updated my PR with a screenshot. Actually, you may be able to answer this. I am trying to add a hover state but I can't seem to change the style of expression value which is a Rep component, so it doesn't look as good yet.

**Comment by jbhoosreddy** [comment]
@jasonLaster I did not yet see it not updating expressions on pause, but I'll keep a look out for it. I did notice sometimes the expression input opens up a new `SOURCEundefined` tab. Also watch expressions does not seem to return a value on Chrome even though the code is executing completely. I should push my latest changes soon.

**Comment by jasonLaster** [comment]
Cool. 

What do you mean by hover?

Chrome won't wont work yet. That's fine. 

Undefined is strange, when do you see that?

**Comment by jbhoosreddy** [comment]
> Chrome won't wont work yet. That's fine.

Oh I didn't know that.

> Undefined is strange, when do you see that?

STR:

1. Enter `document` or `window` in the expression input. 

ER: expression value would be document uri
AR: <not available> and the **debugger is paused**. If you `resume`, the correct expression value is applied.

Here's what I think: If you remember we saw something exactly similar when writing tests with Cypress. I tried to debug the `evaluate` implementation on the debugger server. When the server gets script "document" it returns back a consoleActor with the text "document" which the debugger adds as a source in the sources list. And the debugger is paused. Once you resume it, it goes ahead and emits the result once the console evaluates the script and that propagates back.

While I understood/think I understood the execution of the code, I don't think I can solve it. Would you mind looking at this with me @jasonLaster?

**Comment by jasonLaster** [comment]
@jbhoosreddy yeah - find a time to pair on this issue.

**Comment by jasonLaster** [comment]
@jbhoosreddy, @wldcordeiro I just took another look at this PR and there are some really good things in it:

1. it adds the close button
2. it adds styling for errors
3. it adds the header buttons
4. other good stuff i'm probably missing

the biggest critique i have at the moment is that it sets the input visibility in the redux store and i think it should be local component state. @jbhoosreddy I think i can finish this up if you'd like. or feel free to jump in and take a look. 

> Enter document or window in the expression input

I think this is an edge case we can ignore until we get the basic mechanics in. I have a PR that i'll open soon that fixes some of the redux logic w/ the client.

**Comment by jbhoosreddy** [comment]
I'll make those changes and rebase to clean up this PR, as per your suggestions today.

**Comment by jbhoosreddy** [comment]
I updated the PR with the change as suggested. I left the CSS stuff for now as some work has to be done there.. I'll take that up separately.

**Pull request merged**